pymilvus/orm/tests/utils.py

```diff
@@ -42,23 +43,23 @@ def binary_support():
     return ["BIN_FLAT", "BIN_IVF_FLAT"]
 
 def gen_collection_name():
-    return f'ut-collection-' + str(random.randint(100000, 999999))
+    return f'ut_collection_' + str(random.randint(100000, 999999))
```

Comment:
Why do we need this changes?

Reply:
These names are illegal for containing "-", we only support "_"

Reply:
BTW, the unit tests are still unrunnable, needs further amendment.